DeusEx/Classes/DXRKeys.uc

```diff
@@ -147,12 +147,14 @@ function AdjustRestrictions(int doorspickable, int doorsdestructible, int device
         if( d.KeyIDNeeded != 'None' ) l("found door with KeyIDNeeded == " $ d.KeyIDNeeded);
         //if( d.KeyIDNeeded == 'None' || d.bPickable || d.bBreakable ) continue;
 
-        if( d.bHighlight == false ) {
+        /*if( d.bHighlight == false || d.bFrobbable == false ) {
```

Comment:
there are some strange DeusExMovers out there, some have a Brush that is invisible and I can't figure out how to test if it's invisible or not